.github/pull_request_template.md

```diff
@@ -0,0 +1,41 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Description
+<!--- Describe your changes in detail -->
+<!--- When creating a new PR, make sure to request a review and tag related issues -->
+
+## Motivation and Context
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here. -->
```

Comment:
You are already requesting issues to be linked in the description section. I would only link issues in one place.

Reply:
Ditto above.